rtil/src/threads/ue/rebo/livesplit.rs

```diff
@@ -8,6 +8,15 @@ pub struct LiveSplit {
     pub timer: LiveSplitTimer,
 }
 pub struct Timer {}
+pub struct Run {}
+
+#[derive(rebo::ExternalType)]
+pub enum Games {
```

Comment:
Enum names should generally be singular. See also [rust examples in TRPL](https://doc.rust-lang.org/book/ch06-01-defining-an-enum.html), [Java enum guide](https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html), [C# enum naming](https://learn.microsoft.com/en-us/previous-versions/dotnet/netframework-1.1/4x252001(v=vs.71)?redirectedfrom=MSDN), [Swift](https://docs.swift.org/swift-book/documentation/the-swift-programming-language/enumerations/).